torch/_export/converter.py

```diff
@@ -964,6 +968,19 @@ def convert(self) -> ExportedProgram:
         ep = self.retrace_as_exported_program(
```

Comment:
can we also add a log of the graph before the retrace :> (using `gm.print_readable()`)